src/syn_groups.erl

```diff
@@ -126,20 +118,12 @@ member(Pid, GroupName) ->
 -spec get_local_members(Name :: any()) -> [pid()].
 get_local_members(GroupName) ->
     Node = node(),
-    ets:select(syn_groups_by_name, [{
-        {{GroupName, '$2'}, '_', '_', Node},
-        [],
-        ['$2']
-    }]).
+    [Pid || {_GroupName, Pid, _Meta, PidNode} <- ets:lookup(syn_groups_pids_by_group, GroupName), Node =:= PidNode].
```

Comment:
@hauleth Actually the whole point of this PR was to get rid of `ets:select/2` when retrieving the list of PIDs because of it's poor performance for bigger number of registered processes (requirement to always scan whole table). `ets:lookup/2` execution time is constant for `duplicate_bag`, regardless of the table size.